docs/data/date-pickers/accessibility/accessibility.md

```diff
@@ -0,0 +1,96 @@
+This is the most common target for organizations so it is what we aim to support.
+
+<!-- Those examples do not cover the clock component -->
+
+The WAI-ARIA Authoring Practices includes examples on [Date Picker Dialog](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/datepicker-dialog/) and [Date Picker Spin Button](https://www.w3.org/WAI/ARIA/apg/patterns/spinbutton/examples/datepicker-spinbuttons/) that provide valuable information on how to optimize the accessibility of the Date and Time Pickers.
+
+## Screen reader compatibility
+
+Date and Time Pickers provide support to ARIA roles and a robust focus management across the interactive elements to convey the necessary information to users, being optimized for use with assistive technologies.
+
+## Keyboard support
+
+The Date and Time Pickers consist of different associations of Fields, Calendar, Time and Digital Clock components. Each of these are designed to respond intuitively to keyboard interactions, providing a better experience for users who rely on keyboard navigation.
+
+Each available component presents its own keyboard support, detailed on the following section:
+
+### Fields
+
+The following table describes the keyboard support for all [field components](/x/react-date-pickers/fields/)
+
+|                               Keys | Description                                  |
+| ---------------------------------: | :------------------------------------------- |
+|  <kbd class="key">Arrow Left</kbd> | Moves focus among date/time sections         |
+| <kbd class="key">Arrow Right</kbd> | Moves focus among date/time sections         |
+|    <kbd class="key">Arrow Up</kbd> | Increases focused section value by 1         |
+|  <kbd class="key">Arrow Down</kbd> | Decreases focused section value section by 1 |
+|     <kbd class="key">Page Up</kbd> | Increases focused section value section by 5 |
+|   <kbd class="key">Page Down</kbd> | Decreases focused section value section by 5 |
+|        <kbd class="key">Home</kbd> | Sets focused section to the first value      |
+|         <kbd class="key">End</kbd> | Sets focused section to the last value       |
+
+### Date Calendar
+
+All the available [views](/x/react-date-pickers/date-calendar/#views) implement the same basic keyboard support:
+
+|                                                      Keys | Description                            |
+| --------------------------------------------------------: | :------------------------------------- |
+|                                <kbd class="key">Tab</kbd> | Moves focus to the next element        |
+|   <kbd class="key">Shift</kbd>+<kbd class="key">Tab</kbd> | Moves focus to the previous element    |
+| <kbd class="key">Arrow Keys (Left, Right, Up, Down)</kbd> | Moves focus among the available values |
+
+In addition to the basic keyboard support above, the day view component implements its own keyboard interactions:
```

Comment:
I agree, but I also felt that the keyboard support differences among the views was a relevant information 🤔

Reply:
Yes, that is very relevant information, it's just that `day view` sounds and somewhat is like an internal notion, but I'm fine with going for it now and adjusting if necessary or when we think of something better. 👍

Reply:
I assumed that since we mention it [here](https://mui.com/x/react-date-pickers/date-calendar/#views) it was a known therm for the docs
I'm also not so happy with this part, but I couldn't come with a better reference 😢